### ISSUE TITLE
Missing privacy string

### ISSUE THREAD

**Issue by Syxton** [davosmith/moodle-checklist#54]
privacy:metadata:checklist_comment:item

**Comment by stopfstedt** [comment]
seems obsolete. i only see a reference to the lang string `privacy:metadata:checklist_comment:itemid`, and that one is backed by a translation.

```
$ grep -r 'privacy:metadata:checklist_comment:item' .
./lang/en/checklist.php:$string['privacy:metadata:checklist_comment:itemid'] = 'The ID of the checklist item that the comment relates to';
./classes/privacy/provider.php:                'itemid' => 'privacy:metadata:checklist_comment:itemid',
```

**Comment by davosmith** [comment]
Fixed in commit c485168
